src/components/connect-button.ts

```diff
@@ -46,40 +43,70 @@ export class ConnectButton extends LitElement {
   personaData: PersonaData[] = []
 
   @property({ type: Object })
-  explorer: { baseUrl: string; transactionPath: string; accountsPath: string } =
-    {
-      baseUrl: 'https://betanet-dashboard.radixdlt.com/',
-      transactionPath: 'transaction/',
-      accountsPath: 'account/',
-    }
+  explorer: ExplorerConfig = defaultExplorerConfig
+
+  @property({
+    type: Boolean,
+  })
+  isMobile: boolean = false
+
+  @property({
+    type: Boolean,
+  })
+  isWalletLinked: boolean = false
+
+  @property({
+    type: Boolean,
+  })
+  isExtensionAvailable: boolean = false
+
+  @property({
+    type: Boolean,
+  })
+  fullWidth: boolean = false
+
+  @property({
+    type: String,
+  })
+  status: RadixButtonStatus = RadixButtonStatus.default
+
+  @property({
+    type: String,
+    reflect: true,
+  })
+  activeTab: 'sharing' | 'requests' = 'sharing'
+
+  @property({ type: String })
+  mode: string = ''
 
   @property({ type: String })
   dAppName: string = ''
 
   @property({ type: String })
   personaLabel: string = ''
 
+  @property({ type: String })
+  avatarUrl: string = ''
+
   get hasSharedData(): boolean {
     return !!(this.accounts.length || this.personaData.length)
   }
 
-  private subscriptions = new Subscription()
+  windowClickEventHandler: (event: MouseEvent) => void
 
   private readonly fontGoogleApiHref =
```

Comment:
we can remove this

Reply:
do we assume it'll be always provided by dApp developer?

Reply:
It's imported in CSS now.

Reply:
`src/styles/fonts.css`

Reply:
AFAIR this will cause double font load for people who already have it